ngraph/python/tests/conftest.py

```diff
@@ -43,6 +49,7 @@ def pytest_configure(config):
 
 def pytest_collection_modifyitems(config, items):
     backend_name = config.getvalue("backend")
+    tests.ADDITIONAL_MODELS_DIR = config.getvalue("additional_models")
```

Comment:
Not needed.

Reply:
Needed with configurable path to additional models path.